src/test/java/HandlerGetFileContentsTest.java

```diff
@@ -6,63 +6,63 @@
 public class HandlerGetFileContentsTest extends junit.framework.TestCase {
 
   HandlerGetFileContents handler;
-  ResponseHeadersForContent responseHeadersForContent;
+  ContentTypeForFileExtension contentTypeForFileExtension;
   HttpRequest request;
   HttpResponse response;
   MockFileIO mockFileIO;
 
   protected void setUp() {
     mockFileIO = new MockFileIO();
-    responseHeadersForContent = new ResponseHeadersForContent();
+    contentTypeForFileExtension = new ContentTypeForFileExtension();
     mockFileIO.setIsFileTrue();
     mockFileIO.setIsDirectoryFalse();
   }
 
   public void testGenerateOkResponseDefaultPath() {
     String path = "/";
-    handler = new HandlerGetFileContents(path, mockFileIO, responseHeadersForContent);
+    handler = new HandlerGetFileContents(path, mockFileIO, contentTypeForFileExtension);
     response = handler.generateResponse();
     assertEquals(response.responseCode(), 200);
   }
 
   public void testGenerateOkResponseValidFile() {
     String path = "/something.txt";
-    handler = new HandlerGetFileContents(path, mockFileIO, responseHeadersForContent);
+    handler = new HandlerGetFileContents(path, mockFileIO, contentTypeForFileExtension);
     response = handler.generateResponse();
     assertEquals(response.responseCode(), 200);
   }
 
   public void testGenerateNotFoundResponseforInvalidFile() {
     String path = "/throwIOException";
-    handler = new HandlerGetFileContents(path, mockFileIO, responseHeadersForContent);
+    handler = new HandlerGetFileContents(path, mockFileIO, contentTypeForFileExtension);
     response = handler.generateResponse();
     assertEquals(response.responseCode(), 404);
   }
 
   public void testContentTypeIsTextHtmlForTxtFile() {
     String path = "/something.txt";
-    handler = new HandlerGetFileContents(path, mockFileIO, responseHeadersForContent);
+    handler = new HandlerGetFileContents(path, mockFileIO, contentTypeForFileExtension);
     response = handler.generateResponse();
     assertEquals(response.headers().get("Content-Type"), "text/plain");
   }
 
   public void testContentTypeIsImageJpegForJpg() {
```

Comment:
Since you have the content-type mapping already tested in `ContentTypeForFileExtensionTest`, it's _probably_ safe to rip out a couple of these tests here in `HandlerGetFileContentsTest`.